Vcpkg Android platforms are missing catopen, catgets and catclose functions.

--- a/runtime/src/kmp_i18n.cpp
+++ b/runtime/src/kmp_i18n.cpp
@@ -70,7 +70,7 @@
 } // func __kmp_i18n_catopen
 
 /* Linux* OS and OS X* part */
-#if KMP_OS_UNIX
+#if KMP_OS_UNIX && !__ANDROID__
 #define KMP_I18N_OK
 
 #include <nl_types.h>
